bite-sized-bears/Website/syndication_app/models.py

```diff
@@ -0,0 +1,77 @@
+from django.db import models
+from django.db.models.signals import post_save
+from django.contrib.auth.models import User as AuthUser
+
+
+class User(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=15, unique=True)
+    password = models.CharField(max_length=15)
+    avatar = models.ImageField(upload_to='img/users/avatar', blank=True, null=True)
+    join_date = models.DateTimeField(auto_now=True)
+    parent = models.OneToOneField(AuthUser, on_delete=models.CASCADE, blank=True, null=True)
+
+    def get_absolute_url(self):
+        return '/user/' + str(self.id)
+
+    def __str__(self):
+        return self.name
+
+
+class Community(models.Model):
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=25)
+    description = models.TextField(blank=True, null=True)
+    owner = models.ForeignKey(User, on_delete=models.CASCADE, related_name='community_owner')
+    admin = models.ManyToManyField(User, related_name='community_admin')
+    subscribers = models.ManyToManyField(User, related_name='community_subscribers')
+    location = models.CharField(max_length=20)
+
+    def get_absolute_url(self):
```

Comment:
Using properties in Python with the `@property` decorator is more idiomatic than having getters.